Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -1368,6 +1368,99 @@ final class AppDelegateViewModelTests: TestCase {
     )
   }
 
+  func testVisitorCookies_ApplicationWillEnterForeground() {
+    let existingCookie = HTTPCookie(
```

Comment:
In these other tests I've added an existing cookie to test that setting the visitor cookie multiple times does not affect any other cookies we may have set.